src/user-guide/installing-workflows.rst

```diff
@@ -87,6 +87,59 @@ then ``cylc install dogs/fido`` will search for a workflow source directory
 ``~/cylc-src/dogs/fido``, or, failing that, ``~/roses/dogs/fido``, and install
 the first match (into ``~/cylc-run/dogs/fido/run1``).
 
+
+.. _SymlinkDirs:
+
+Symlink Directories
+^^^^^^^^^^^^^^^^^^^
+
+You can configure workflow :term:`run directories <run directory>` and certain
+sub-directories as symlinks to other locations. This is a useful way of
+offloading data onto other drives to limit the disk space taken up by
+``~/cylc-run``.
+
+Directories that can be individually symlinked are:
+
+* ``log``
+* ``share``
+* ``share/cycle``
+* ``work``
+* the :term:`run directory` itself
+
+They are configured in :cylc:conf:`global.cylc[install][symlink dirs]`.
+
+Cylc will automatically create these symlinks on install;
+see :ref:`Symlinking of directories`.
```

Comment:
If you decide to keep the numbers https://github.com/cylc/cylc-doc/pull/369/files#r781123840 you might want to do this to prevent the number actually appearing here.
```suggestion
see :ref:`Symlinking of directories <Symlinking of directories>`.
```